locality-checker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.14-alpine3.11 as builder
+
+RUN apk add --no-cache ca-certificates make gcc libc-dev yarn git libc6-compat curl
```

Comment:
We may need to consider a builder image for this tool so that the final image does not bring all unnecessary packages.